### PR TITLE
Update agent connection for enabling multi-cluster mgmt

### PR DESCRIPTION
**Pull request by rayzhou2017**
Signed-off-by: Ray Zhou <ray@yunify.com>

### CODE REVIEW

File: content/en/docs/multicluster-management/enable-multicluster/agent-connection.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ multicluster:
 
 {{< tab "KubeSphere has not been installed" >}}
 
-There is no big difference than installing a standalone KubeSphere if you define a host cluster before installation. Please note that the `clusterRole` in `config-sample.yaml` or `cluster-configuration.yaml` has to be set as follows:
+There is no big difference than installing a standalone KubeSphere if you define a host cluster before installation. Please note that the `clusterRole` in `config-sample.yaml` has to be set as follows:
```
Comment (Sherlock113) [inline content/en/docs/multicluster-management/enable-multicluster/agent-connection.md:50]: I have a question here. Why `cluster-configuration.yaml` is deleted here? My understanding is that If users already have a Kubernetes cluster, they can set `clusterRole` to `host` in `cluster-configuration.yaml` before installation. After they apply the file to install KubeSphere, the cluster will be the host cluster.

File: content/en/docs/multicluster-management/enable-multicluster/agent-connection.md [hunk 2]
```diff
@@ -180,7 +181,7 @@ multicluster:
 
 {{< tab "KubeSphere has not been installed" >}}
 
-There is no big difference than installing a standalone KubeSphere if you define a member cluster before installation. Please note that the `clusterRole` in `config-sample.yaml` or `cluster-configuration.yaml` has to be set as follows:
```
Comment (Sherlock113) [inline content/en/docs/multicluster-management/enable-multicluster/agent-connection.md:183]: Same question here.

**Comment by Sherlock113** [comment]
Another thing is that technically, the **Prerequisites** section of both Agent Connection and Direct Connection is both not accurate. It says "You have already installed at least two KubeSphere clusters.", while these two guides both have the part "KubeSphere has not been installed" if users want to prepare a host cluster. This is quite strange because if the prerequisites of having two KubeSphere clusters are met, the condition of "KubeSphere has not been installed" does not exist. However, to prepare a member cluster, a host cluster is a must as the jwtsecret is needed. I will refactor both Direct Connection and Agent Connection later.

**Comment by rayzhou2017** [comment]
@Sherlock113 Please review the new change to see if it is clear now.

**Pull request merged**